bemani/backend/jubeat/base.py

```diff
@@ -64,7 +64,7 @@ def format_profile(self, userid: UserID, profile: Profile) -> Node:
         """
         return Node.void('gametop')
 
-    def format_scores(self, userid: UserID, profile: Profile, scores: List[Score]) -> Node:
+    def format_scores(self, userid: UserID, profile: Profile, scores: List[Score], mdata_ver: Optional[int]) -> Node:
```

Comment:
I would call this "page" or "part" or something more akin to what it does, instead of just what the game calls it. The game's name for this is about the worst name for something I've ever seen. Maybe even "chunk", not sure, but the current name doesn't make sense in any respect. Obviously the name should go in the other function prototypes as well.